src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1502,6 +1502,51 @@ public static IEnumerable<string> ParseConditionalCompilationSymbols(string valu
             return defines.AsEnumerable();
         }
 
+        public static bool TryParseLanguageVersion(string value, out LanguageVersion version)
```

Comment:
I guess my only real concern is @jcouv's change is targeting master, and I'm not sure I'm comfortable with that being merged but your fix to the project system _not_ being merged. Perhaps you and him should migrate your change to remove the IDE's parsing function and use the compiler API over to his?